tests/test_internet_iface_sel.py

```diff
@@ -0,0 +1,73 @@
+import sys
```

Comment:
We don't need a separate file for all the stuff you added. Just add them to the same test class as the rest of the `NetworkManager` methods.